static/js/components/allocation/AllocationSummary.jsx

```diff
@@ -105,7 +105,7 @@ const SimpleMenu = ({ request }) => {
         open={Boolean(anchorEl)}
         onClose={handleClose}
       >
-        {(request.status === "submitted" ||
+        {(request.status.includes("submitted") ||
```

Comment:
same comment here, we want to check that it starts with submitted, not just that it's present somewhere (could be in an error msg)